src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyCultureProvider.cs

```diff
@@ -12,6 +13,7 @@
 
 namespace Microsoft.AspNetCore.Components.WebAssembly.Hosting
 {
+    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026", Justification = "This type loads resx files. We don't expect it's dependencies to be trimmed in the ordinary case.")]
```

Comment:
I admit I don't fully understand what this does, but the justification sounds like "we hope this will work". That's typically not good enough. Maybe we need a different solution here?

Reply:
Blazor WebAssembly is responsible for it's own assembly loading. With resx files, it waits until after the app has started `Program.MainAsync` has executed to load satellite assemblies based on current culture. These resource assemblies are part of the initial set of assemblies that are trimmed, so we're not loading any assemblies that hadn't participated during linking.